easybuild/easyconfigs/l/libsamplerate/libsamplerate-0.1.9-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsamplerate'
+version = '0.1.9'
+
+homepage = 'http://www.mega-nerd.com/libsamplerate'
+description = "Secret Rabbit Code (aka libsamplerate) is a Sample Rate Converter for audio."
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://www.mega-nerd.com/libsamplerate/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a7eb168e2f21353fb6d84da152e4512126f7dc48ccb0be80578c565413444c1']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    # FFTW is only required for tests
+    ('FFTW', '3.3.8', '', ('gompi', '2019a')),
```

Comment:
is this supposed to work with an hierarchical MNS?

Reply:
Hmm, good question, I may have to also include `gompi` separately as dependency...

@Micket You have an HMNS setup, right? Can you submit a test report for this PR?